lib/karmap/watchdog.rb

```diff
@@ -374,5 +380,36 @@ def sync_services_statuses
       end
       @service_statuses = new_service_statuses
     end
+
+    #
+    # Send a notification to AirBrake (using HTTP Api)
+    # use ENV['AIRBRAKE_KARMA_PROJECT_ID'] and ENV['AIRBRAKE_KARMA_PROJECT_KEY']
+    #
+    # @param [String] notification: message of the notification
+    # @return [Bool] true if notification is sent (with success), false otherwise
+    #
+    def send_airbrake_notification(notification_type = 'notification')
```

Comment:
sync parameters with documentation
the method seems to require notification param
check also the call (raw: 118)